org.eclipse.ice.modeling/src/main/java/org/eclipse/ice/modeling/workflowEngine/Step.java

```diff
@@ -0,0 +1,56 @@
+package org.eclipse.ice.modeling.workflowEngine;
+
+public class Step {
+
+	private Action action;
```

Comment:
If a step has one action, what is differentiating it from an action other than the SuccessCriteria? Is Step just a stateful implementation of Action? I need to think through this also.

Reply:
Good question and not sure yet.  Need to have a better idea of what the processing is for this stuff and run some scenarios.  In the abstract though we need to keep and use the same structure if a step has one action or several.  I actually see an action as being fairly atomic, at least from a workflow standpoint.  Otherwise it kind of gets into being another layer of workflow.  I am totally unclear at this point on much of this and need to work through with you.  I am not clear what all state even needs to be captured at the Step (Task) level.  My gut tells me there is and that is supported by Jay's ontology, but I am way lacking in real detail and understanding at this point.